runtime/onert/core/src/compiler/Linear.cc

```diff
@@ -56,7 +56,7 @@ std::vector<ir::OpSequenceIndex> Linear::linearize(const ir::LoweredGraph &lower
     //            |
     //           [4]
     op_seqs.iterate([&](const ir::OpSequenceIndex &op_seq_idx, const ir::OpSequence &op_seq) {
-      for (auto input : op_seq.getInputs())
+      for (auto input : op_seq.getInputs().asUnique())
```

Comment:
I didn't see this change in the previous change and could you explain why we need this? I thought it worked fine without `asUnique`.

Reply:
It was unordered_set in the previous change.
Right. In this block it worked fine without `asUnique`. But if op_seq has some inputs as the same `Operandindex`, the list(`std::list<ir::OpSequenceIndex> list{op_seq_idx};`) below will have dulplicated indices.